### DISCUSSION TITLE
Future of starboard - migrate starboard to trivy-operator?

### DISCUSSION THREAD

**Discussion by skuethe** [aquasecurity/starboard#1169] [Questions & Help] [answered]
Hey there,

I have developed a strong interest in this project, so I am very curious about the just opend issues to rename this project to `trivy-operator`:
https://github.com/aquasecurity/starboard/issues/1167

Can you give any insights to this?
Does this mean what the name implies, and the functionality will be reduced to trivy functionality only?

Thank you in advance!

**Comment by skuethe** [comment, marked as answer]
There was feedback on the slack channel:

> We will be sharing more details and official announcements to the community over the next two days

**Reply by nissessenap** [reply]
I guess the discussion will continue in https://github.com/aquasecurity/starboard/discussions/1173
